app/views/race_editions/enter.html.erb

```diff
@@ -29,6 +29,13 @@
       <%= racer_fields.label :state %>
       <%= racer_fields.select :state, StatesHelper::STATES  %>
     <% end %>
+
+    <%= race_edition_form.fields_for :race_entries, @race_entry do |race_entry_fields| %>
```

Comment:
I don't completely understand this line. Is it looping over one field? Predicted time? But designed to handle more fields? I'm sure most of my confusion is over Rails-isms.